server/coordinator.go

```diff
@@ -368,7 +376,27 @@ func (c *coordinator) collectHotSpotMetrics() {
 			hotSpotStatusGauge.WithLabelValues(storeAddress, storeLabel, "hot_read_region_as_leader").Set(0)
 		}
 	}
+}
 
+func (c *coordinator) resetHotSpotMetrics() {
+	c.RLock()
+	defer c.RUnlock()
+	// Resets hot write region metrics.
+	stores := c.cluster.GetStores()
+	for _, s := range stores {
+		storeAddress := s.GetAddress()
+		storeID := s.GetID()
+		storeLabel := fmt.Sprintf("%d", storeID)
+		// Resets hot write metrics
+		hotSpotStatusGauge.WithLabelValues(storeAddress, storeLabel, "total_written_bytes_as_peer").Set(0)
```

Comment:
Can we get all the `label values`? It's easy to ignore some clean up when we add new metrics. It seems the` Desc()` can get the `LabelPair`.